ci/env/install-java.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+# Push caller's shell options (quietly)
+{ SHELLOPTS_STACK="${SHELLOPTS_STACK-}|$(set +o); set -$-"; } 2> /dev/null
+
+set -euxo pipefail
+
+SCRIPT_DIR=$(builtin cd "$(dirname "${BASH_SOURCE:-$0}")"; pwd)
+WORKSPACE_DIR="${SCRIPT_DIR}/../.."
+
+sudo apt-get install -y maven openjdk-8-jre openjdk-8-jdk
+"${WORKSPACE_DIR}"/java/build-jar-multiplatform.sh linux
```

Comment:
Do we need to build the Ray java jars? We don't run any java code, right?

Reply:
Raydp create java actors. Such as [here](https://github.com/oap-project/raydp/blob/master/core/raydp-main/src/main/java/org/apache/spark/deploy/RayAppMasterUtils.java#L32)